config/prow/plugins.yaml

```diff
@@ -72,7 +72,7 @@ approve:
   - kubernetes-client
   - kubernetes-csi
   - kubernetes-sigs
-  require_self_approval: false
+  require_self_approval: true
```

Comment:
you probably don't mean to be changing this for `kubernetes/kubernetes`, every repo in `kubernetes-sigs`, etc.

move `kubernetes/kops` to the entry above that contains `kubernetes/website`

Reply:
oups. I most certainly did not.

Thanks. Moved.